oe_webtools_analytics/oe_webtools_analytics.info.yml

```diff
@@ -0,0 +1,8 @@
+name: OpenEuropa Webtools Analytics
+description: Provides the configuration settings for webtools analytics integration.
+package: OpenEuropa Webtools
```

Comment:
I guess the package name must stay "OpenEuropa Webtools" as this module has submodules ?
If this is the case, then my previous comment about the package can be discarded.

Reply:
Yes, it has!